src/div_fns.bak

```diff
@@ -0,0 +1,184 @@
+
+/// Returns 0 in `r0`, while placing the `numerator` into `r1`.
+///
+/// This is written in that slightly strange way so that `div` function and
+/// `divmod` functions can share the same code path.
+///
+/// See: [__aeabi_idiv0][aeabi-division-by-zero]
+///
+/// [aeabi-division-by-zero]: https://github.com/ARM-software/abi-aa/blob/main/rtabi32/rtabi32.rst#division-by-zero
+#[naked]
```

Comment:
Do you intend to change these `#[naked]` functions as well?

Reply:
Yeahhhhh i probably should.

The thing is that compiler_builtins can provide division with these missing, and if you're using enough division that you care to have a version that's optimized for whatever speed/size balance and put in iwram and stuff  then probably that programmer knows more about the right fn to use than me the crate author does.

When i did some simple benchmarking there were faster division functions than the ones here, but none of the faster ones had as small a code size in memory, so there's a real choice to be made for those who care about the difference.

Reply:
I decided to just not. We can go back to a 0.11 git tag and bring them back if we need them later.